tests/test_ts_mget_cme.py

```
@@ -300,3 +300,95 @@ def test_mget_cme_label_consistency(self):
+
+    def test_mget_cme_hashtag_with_other_options(self):
+        """HASHTAG composes with the other TS.MGET options, in any position"""
+        cluster_client: ValkeyCluster = self.new_cluster_client()
+        tags, keys = self.setup_tagged_data(cluster_client)
+
+        client = self.new_client_for_primary(0)
+
+        result = client.execute_command('TS.MGET', 'WITHLABELS', 'HASHTAG', tags[1], 'FILTER', 'name=tagged')
+        assert len(result) == 1
+        assert result[0][0] == keys[1]
+        assert {l[0]: l[1] for l in result[0][1]} == {b'name': b'tagged', b'shard': b'1'}
```

Ruff (0.16.3)

[error] Ambiguous variable name: `l`

(E741)